core/esmf-aspect-meta-model-python/tests/integration/resources/com.bosch.test.models/0.2.0/AspectWithProperties.ttl

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH, Germany. All rights reserved.
+#
+
+@prefix : <urn:samm:com.bosch.test.models:0.2.0#>.
+@prefix type: <urn:samm:com.bosch.test.type:0.2.0#> .
```

Comment:
If this needs to be in another namspace than the default test namespace, it should be included in it, e.g. `urn:samm:org.eclipse.esmf.test.other:1.0.0#`